Spring_Backend/src/main/java/polsl/tab/skiresort/api/technical_employee/TechnicalEmployeeApi.java

```diff
@@ -0,0 +1,24 @@
+package polsl.tab.skiresort.api.technical_employee;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import polsl.tab.skiresort.api.technical_employee.request.SkiLiftRequest;
+import polsl.tab.skiresort.api.technical_employee.response.SkiLiftResponse;
+import polsl.tab.skiresort.api.technical_employee.service.TechnicalEmployeeService;
+
+@RestController
+@RequestMapping("/api/technical_employee")
+public class TechnicalEmployeeApi {
+
+    private final TechnicalEmployeeService technicalEmployeeService;
+
+    public TechnicalEmployeeApi(TechnicalEmployeeService technicalEmployeeService){this.technicalEmployeeService = technicalEmployeeService;}
+
+    @PutMapping("{skiLiftId}")
+    public ResponseEntity<SkiLiftResponse> editSkiLiftIsOpened(@PathVariable("skiLiftId") Integer skiLiftId,
+                                                             @RequestBody SkiLiftRequest request
```

Comment:
There is no need of passing whole SkiLiftRequest, we only change state true/false. You can do it by id